src/components/App/App.js

```diff
@@ -37,22 +38,36 @@ class App extends React.Component {
             <Route
               exact
               path="/areas"
-              render={() => <AreasContainer user={this.state.userName}/>}
+              render={() => <AreasContainer
+                user={this.state.userName}
```

Comment:
Per above comment, we could avoid the need for multiple props here.

Reply:
I will make the change to not include multiple props

Reply:
Changed to use one props